### PR TITLE
Chore/2410 update

### PR DESCRIPTION
**Pull request by jrconlin**
## Description

Update py03

### CODE REVIEW

File: tokenserver-auth/src/oauth/py.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ use std::{sync::Arc, time::Duration};
 pub struct Verifier {
     // Note that we do not need to use an Arc here, since Py is already a reference-counted
```
Comment (pjenvey) [inline tokenserver-auth/src/oauth/py.rs:22]:
```suggestion
```

and note to myself: [Pyo3's migration guide](https://pyo3.rs/main/migration#pyclone-is-now-gated-behind-the-py-clone-feature) recommended moving to Arc<Py<>>

**Review by pjenvey — CHANGES_REQUESTED** [review]
thanks for making this very limited in scope but I might wait till we get the tokenserver sentry client bits sorted out/released before merging this anyway

File: tokenserver-auth/src/oauth/py.rs (hunk 1 above)
Comment (jrconlin) [reply tokenserver-auth/src/oauth/py.rs:22]: Thanks! forgot to clean up that comment.

**Comment by jrconlin** [comment]
Yeah, zero rush on landing this. I figured I'd just do it quick while it was top of mind.

**Review by pjenvey — APPROVED** [review]

**Pull request merged**